.travis.yml

```diff
@@ -10,7 +10,7 @@ script:
 deploy:
 - provider: script
   skip_cleanup: true
-  script: npx semantic-release
+  # script: npx -p @qiwi/semrel-toolkit multi-semrel
```

Comment:
Will uncomment this before merging, just didn't want to make any unintentional moves before everything is in place